pkg/csi/cinder/controllerserver.go

```diff
@@ -83,21 +95,11 @@ func (cs *controllerServer) CreateVolume(ctx context.Context, req *csi.CreateVol
 	}
 	volSizeGB := int(util.RoundUpSize(volSizeBytes, 1024*1024*1024))
 
-	// Volume Type
-	volType := volParams["type"]
-
-	var volAvailability string
-	if cs.Driver.withTopology {
-		// First check if volAvailability is already specified, if not get preferred from Topology
-		// Required, incase vol AZ is different from node AZ
-		volAvailability = volParams["availability"]
-		if volAvailability == "" {
-			accessibleTopologyReq := req.GetAccessibilityRequirements()
-			// Check from Topology
-			if accessibleTopologyReq != nil {
-				volAvailability = sharedcsi.GetAZFromTopology(topologyKey, accessibleTopologyReq)
-			}
-		}
+	// Volume cloud
+	cloud, cloudExist := cs.Clouds[volCloudAvailability]
```

Comment:
I thought `volCloudAvailability` would return a region (e.g. `regionOne`). Why are we treating it like a cloud entry here (e.g. `devstack-admin`)?